mhubert-pmn/Front-Url-Shortener
# [Update] Update input style

## Task
Update input style

## Type of change

## Description
- Update input style
- Fix handleChange function
- Remove isRequired for the prop type onChange
- Fix form components folder name

## Testing scenario
In the terminal:
- Launch and build: `docker-compose up --build
- Go to `http://localhost:3000/inscription` and check the style of the input
- Change the size of your screen to see the responsive style

## Trello card
[https://trello.com/c/9k85kONU](https://trello.com/c/9k85kONU)
